src/components/WelcomeToSDC.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import {
+    Box,
+    HStack,
+    VStack,
+    Heading,
+    Text,
+    Image,
+    Button,
+    Center,
+    Flex,
+    Grid,
+    GridItem,
+} from "@chakra-ui/react";
+import { Tabs, TabList, TabPanels, Tab, TabPanel } from "@chakra-ui/react";
+
+export const WelcomeToSDC: React.FC = () => {
+    // TODO remove test data and get new images
+    const title = "Welcome to SDC";
+    const desc1 = "Registration for Summer 2021 classes begins June 31, 2021!";
+    const text1 =
+        "Browse through programs below and then select the day/time preferred! We make sure that 100% of our participants' demonstrated needs are met.";
+    const text2 =
+        "For volunteers, create an account to volunteer for SDC and then select the program you want to volunteer for!";
+    const img1 =
+        "https://img.webmd.com/dtmcms/live/webmd/consumer_assets/site_images/article_thumbnails/other/active_kids_other/1800x1200_active_kids_other_alt.jpg";
```

Comment:
Will these images be replaced? If so, can you leave a comment. For images that we want, I think we should download and store them ourselves instead of using a link.